standard/standard_1.php

```diff
@@ -960,16 +961,37 @@ function trim (string $string, string $characters = " \t\n\r\0\x0B"): string
 function ltrim (string $string, string $characters = " \t\n\r\0\x0B"): string
 {}
 
+#[PhpStormStubsElementAvailable(to: '7.3')]
+/**
+ * Strip HTML and PHP tags from a string
+ * @link https://php.net/manual/en/function.strip-tags.php
+ * @param string $string <p>
+ * The input string.
+ * </p>
+ * @param null|string $allowed_tags [optional] <p>
+ * You can use the optional second parameter to specify tags which should
+ * not be stripped.
+ * </p>
+ * <p>
+ * HTML comments and PHP tags are also stripped. This is hardcoded and
+ * can not be changed with allowable_tags.
+ * </p>
+ * @return string the stripped string.
+ */
+#[Pure]
```

Comment:
Seems here also `LanguageLevelTypeAware` can be used instead of function duplication